src/EA.Weee.Web.Tests.Unit/Areas/AatfReturn/Attributes/ValidateReturnCreatedActionFilterAttributeTests.cs

```diff
@@ -34,14 +34,22 @@ public ValidateReturnCreatedActionFilterAttributeTests()
         [Fact]
         public async void OnActionExecuting_GivenReturnStatusIsNotCreated_ShouldBeRedirectedToTaskList()
         {
-            var returnData = new ReturnStatusData()
+            var returnStatusData = new ReturnStatusData()
             {
                 OrganisationId = Guid.NewGuid(),
                 ReturnStatus = ReturnStatus.Submitted
             };
 
+            var returnData = new ReturnData()
+            {
+                QuarterWindow = new QuarterWindow(new DateTime(DateTime.Now.Year, 01, 01), new DateTime(DateTime.Now.Year, 03, 31))
```

Comment:
This will fail when the tests are run in Jan/Feb/Mar next year. You'll need to use a specific date and SystemTime.Freeze() instead

Reply:
Good spot, use SystemTime in code where possible